release/nightly_tests/stress_tests/test_placement_group.py

```diff
@@ -127,7 +128,8 @@ def pg_launcher(pre_created_pgs, num_pgs_to_create):
             total_removing_time += (end - start)
 
     # Validate the correctness.
-    assert ray.cluster_resources()[
+    time.sleep(10)
```

Comment:
do we need this?

Reply:
Yeah. It takes some time to clean it up. Probably not the robust logic, but let's see

Reply:
reverted for now since it will be handled by other PR